Lessons8/HomeWork.py

```diff
@@ -0,0 +1,52 @@
+"""
+HomeWork lesson 8.
+
+Given the list of integers ( positive, negative, zeros)
+Find max multiplication of three values in list
+
+Input: [-1, 1, 2, 0, 3, 12, 4, 5, -1, 6]
+Output: Max value = "360". Nums are: (12, 5, 6)
+
+My solution:
+1) Sort the list by the absolute value of each element
+2) Create positive and negative lists
+3) Append values to positive and negative lists
+4) Find the max value and save elements in output
+"""
+l1 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+# Sort the list by absolute value of each element
+l1.sort(key=lambda x: abs(x), reverse=True)
+
+
+# Separate positive and negative elements
+positive, negative = [el for el in l1 if el > 0], [el for el in l1 if el <= 0]
```

Comment:
In general solution works as expected but too complicated. 
As advanced level try to find something more elegant. For example after list sort - pay attention what multiplication patterns could be used , what elements and what indexes involved into these multiplications to get the maximum